src/webgl/3d_primitives.js

```diff
@@ -10,6 +10,179 @@ import p5 from '../core/main';
 import './p5.Geometry';
 import * as constants from '../core/constants';
 
+/**
+ * Starts creating a new p5.Geometry. Subsequent shapes drawn will be added
```

Comment:
Can we add some information here about why and when a user would want to use this function? I think without context it might be unclear to a new user what this feature is for.